automation/README.md

```diff
@@ -0,0 +1,77 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
```

Comment:
You can also consider adding the link to the nice UI with blueocean, though its usually more slow to load:
http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/

Reply:
When I access this url you are motioning, it redirects me to the one which I have there, with this `activity` suffix. Maybe something is wrong on my side.